app/posts/manage-teacher-training-applications/2021-02-25-filter-by-subject.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Filter applications by subject
+description: Let users filter the applications list by the course’s subject
+date: 2021-02-25
+---
+
+We introduced a subject filter to allow users to filter by the course’s subject in the applications list.
+
+## Why we are doing this
+
+Providers need to be able to filter their applications by subject.
+
+When providers share applications with people who do the shortlisting (for example, subject leads in a school or a subject tutor at an HEI), those users only need to see specific subjects and not the entire list.
+
+## An analysis of the course data
+
+From the course data for ‘synced’ providers, there are approximately 40 subjects across primary and secondary levels.
+
+The number of subjects offered by providers in the form of a course varies between providers.
+
+- The average number of subjects offered: 10
+- The maximum number of subjects offered: 28
+- The minimum number of subjects offered: 1
+
+Providers are able to rename subjects when creating a course, whilst retaining the underlying subject code. For example, ‘Primary’ (code: 00) is often renamed to ‘Primary (5-11)’ or ‘Primary (7-11)’. There are also instances where ‘Primary’ is renamed to a subject that already exists, such as ‘Primary with mathematics’ (code: 03).
+
+Some courses are compound subjects—they are made up of several subjects. For example, Biology, Chemistry and Physics together form a course often labelled ‘Combined science’. French, German and Spanish together may form a course labelled ‘Modern languages’.
+
+Other example compound courses include:
+
+- Biology and Physical Education
+- Business Studies and Economics
+- English and Drama
+- Maths and Physics
+
+## What we changed and why
```

Comment:
Check and use tense from previous design post on automatically declining offers.